### PR TITLE
Adds a Pull Request tempalte for CLI

### PR DESCRIPTION
**Pull request by reznord**
Adds a default template for Pull Requests (which we had one before for Issues)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->
+
+**What kind of change does this PR introduce?**
+
+<!-- E.g. a bugfix, feature, refactoring, build related change, etc… -->
+
+**If relevant, did you add tests for your changes?**
```
Comment (prateekbh) [inline .github/PULL_REQUEST_TEMPLATE.md]: arent tests always relevent 😛

Comment (prateekbh) [inline .github/PULL_REQUEST_TEMPLATE.md]: arent tests always relevent

**Comment by reznord** [comment]
Well, I hardly write tests for any of my PRs lol

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+
+**If relevant, link to documentation update:**
```
Comment (prateekbh) [inline .github/PULL_REQUEST_TEMPLATE.md]: not sure how many times'll really get this

File: .github/issue_template.md [hunk 3]
```diff
@@ -3,15 +3,15 @@
...
 
 <!-- A great way to do this is to provide your configuration via a GitHub gist. -->
 
-**What is the expected behavior?**
+**What is the expected behaviour?**
 
-**If this is a feature request, what is motivation or use case for changing the behavior?**
+**If this is a feature request, what is motivation or use case for changing the behaviour?**
 
 **Please mention other relevant information.**
```
Comment (prateekbh) [inline .github/issue_template.md]: can we add browser name and version?

Comment (reznord) [reply .github/issue_template.md]: that's mentioned in the comment below.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (reznord) [reply .github/PULL_REQUEST_TEMPLATE.md]: oh right! we only have one documentation. this is not a valid one (removed)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (reznord) [reply .github/PULL_REQUEST_TEMPLATE.md]: true, changed in the latest commit.

**Comment by reznord** [comment]
Wondering how come it is open for over a week 😂

**Pull request merged**